src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -192,7 +267,7 @@ bool
 AmendmentTableImpl::unVeto (uint256 const& amendment)
 {
     ScopedLockType sl (mLock);
-    AmendmentState* s = getCreate (amendment, false);
+    AmendmentState* s = getExisting (amendment);
```

Comment:
I agree - much cleaner. For future reference, please prefer `auto s = getExisting (amendment);`